### ISSUE TITLE
PaperMC 1.16.4: How do I retain formatting of my motd in server.properties

### ISSUE THREAD

**Issue by VimaMT** [kangarko/ChatControl-Pro#2165]
I have had this issue each time I have tried to install ChatControl Pro during the last few weeks.

I install it, and it modifies my server.properties. It adds a server-name.  The docs make it sound like server-name is for Bungee. I don't run Bungee ... just a straight server.

When I restart my server, my MOTD on the client screen is all wacky.

My MOTD:

motd=\u00C2\u00A7e\u00C2\u00A7lSomewhat Vanilla with a few Toys\u00C2\u00A7r\n(ver 1.16.4 PaperMC)

See attached file for what I see in the client. I have characters in front of the motd and behind it that should not be there. I wiped out the server name in the image so that it would not be considered advertisement. I shortened the server-name, and that did not help.

I do not see any docs telling me what to do in the event that I do not run Bungee.
![Bad Server MOTD](https://user-images.githubusercontent.com/46653361/100524107-e7395f00-317a-11eb-8efa-a7925a7a918d.JPG)

**Comment by VimaMT** [comment]
I have owned a copy of ChatControl for months, but I have only had these issues for the last few weeks. It may have begun around the time you started adding a server-name back into server.properties.   I notice that ChatControl also rearranges some of the other lines in that server.properties file.

I installed ChatControl on a test server. The MOTD seemed alright there. So, line-by-line, I copied the relevant values from server.properties on the test server over to server.properties on what I hope to retain as a production server. It is strange, but I do not see this same issue now. I have the same MOTD now that I had before.

I will restart the server a few times and see if anything changes again.
